hibernate-core/src/main/java/org/hibernate/query/sqm/sql/SqmToSqlAstConverter.java

```diff
@@ -34,8 +33,6 @@ public interface SqmToSqlAstConverter extends SemanticQueryWalker<Object>, SqlAs
 
 	SqmQueryPart<?> getCurrentSqmQueryPart();
 
-	SqmStatement<?> getCurrentSqmStatement();
```

Comment:
I've removed this method from the interface as it was just recently added in https://github.com/hibernate/hibernate-orm/pull/6654 with the sole purpose of this group by path check, and after my change it would be unused.

Anyway, If we want to keep it (since it's an API) I will just restore it.